### DISCUSSION TITLE
Using Stripe-payment-plugin in multi-vendor-plugin

### DISCUSSION THREAD

**Discussion by bimbistyle** [vendurehq/vendure#2793] [Q&A]
Hello everyone, 
this is my first post in this community. Up until now, we've always been able to help ourselves thanks to the excellent documentation provided by Vendure. Many thanks for that. Anyway, we're currently developing a platform based on Vendure. We're using the Multi-Vendor plugin and would like to integrate Stripe Connect. We've taken a closer look at the Payments plugin for Stripe and are facing a challenge. How can we integrate the Stripe Payments plugin into the Multi-Vendor plugin? Since the Multi-Vendor plugin requires a payment handler, we're currently assuming that we need to implement a new payment handler specifically for Stripe Connect. Or is there already a solution to use "Stripe Connect" with the help of the two mentioned plugins and combine them?

**Comment by michaelbromley** [comment]
Hi,

I'm happy to hear the docs have served you well so far! We put a lot of effort into that.

Right now there is no open source Stripe Connect plugin that I know of, so you'll need to implement the integration yourself.
